projects/angular-d3-cloud/src/lib/angular-d3-cloud.component.ts

```diff
@@ -79,8 +80,8 @@ export class AngularD3CloudComponent implements OnChanges, OnInit {
           .style('font-size', d => `${d.size}px`)
           .style('font-family', this.font as any)
           .style('fill', (word, i) => {
-            if (this.autoFill) {
-              return fill(i.toString())
+            if (this.autoFill && this.fillMapper) {
+              return this.fillMapper(word, i)
```

Comment:
This only gets executed if `autoFill` and `fillMapper` has value, and it makes `autoFill` and `fillMapper` tie together. Can we make it like this:
- both are optional, but if `autoFill` is `true` and `fillMapper` doesn't have value (null | undefined | false), then we return normal `fill(i.toString())`
- if both have value then we return `this.fillMapper(word, i)`

Reply:
Yes, I had initialized `fillMapper `with `defaultFillMapper` that is exaclty `fill(index.toString());` but you are right, someone can pass null, undefined or false so it is better your suggestion.